### ISSUE TITLE
[Request] toLowerCase condition keywords

### ISSUE THREAD

**Issue by braco** [mixmaxhq/search-string#21]
`Foo:bar` === `foo:bar` in most cases I can think of. 

Maybe an option {toLowerCase: true} that defaults to true?

**Comment by bradvogel** [comment]
Yeah, or `ignoreCase: true` that defaults to true (would require a major version bump)

**Comment by mericsson** [comment]
Makes sense, good suggestion. I think change:

```js
static parse(str, transformTextToConditions = []
```
https://github.com/mixmaxhq/search-string/blob/8eed146/src/searchString.js#L30

To:
```js
static parse(str, options = {
  // Default options.
  transformTextToConditions: [],
  ignoreOperatorCase: true  // Ignores case of operator only.
});
```

And it would be a major version bump.

What do you think @braco? Want to take a stab at it with a PR?
